gaskij/rpicampusmap
# Restructure API Folder

## Description
api/src folder was tedious to navigate at times. If developer wants to work on CAS, they had to open the and routes/ controllers/ folder separately. It makes more intuitive sense to keep files that are related to one thing together.

Fixes #157 

## Solution
Each "object" such as CAS, Location, info has  their own folder each containing the corresponding controller and route files.

## Known Issues
(Are there any new issues caused or exposed by your changes, or things you would like clarified before merging?)

## Testing Procedures
Check the general functionality of the changes deployed on Heroku. One issue that occurred the first time I tested was not being able to use CAS, but upon a second try it was successful (could have been an issue on CAS side). 

## Checklist for author
